fedmsg.d/endpoints.py

```diff
@@ -29,15 +29,15 @@
         # infrastructure's bus.  Cool, right?  :)
         "fedora-infrastructure": [
             "tcp://hub.fedoraproject.org:9940",
-            #"tcp://stg.fedoraproject.org:9940",
+            # "tcp://stg.fedoraproject.org:9940",
         ],
 
         # For other, more 'normal' services, fedmsg will try to guess the
         # name of it's calling module to determine which endpoint definition
         # to use.  This can be overridden by explicitly providing the name in
         # the initial call to fedmsg.init(...).
-        #"bodhi.%s" % hostname: ["tcp://127.0.0.1:3001"],
-        #"fas.%s" % hostname: ["tcp://127.0.0.1:3002"],
-        #"fedoratagger.%s" % hostname: ["tcp://127.0.0.1:3003"],
+        # "bodhi.%s" % hostname: ["tcp://127.0.0.1:3001"],
+        # "fas.%s" % hostname: ["tcp://127.0.0.1:3002"],
+        # "fedoratagger.%s" % hostname: ["tcp://127.0.0.1:3003"],
```

Comment:
This type of changes always make me wonder, they are not per-say comments, they are lines of code that are commented out. Most often in those situation I ignore pep8, as it would require two actions to get these lines back working.

Just a personal preference though